numcosmo/math/ncm_stats_dist_nd_kde_studentt.c

```diff
@@ -1031,14 +1075,13 @@ _ncm_stats_dist_nd_kde_studentt_eval_m2lnp (NcmStatsDistNd *dnd, NcmVector *y)
       m2lnp_i += gsl_pow_2 (ncm_vector_fast_get (row_i, k) - ncm_vector_fast_get (self->v, k));
     }
 
-    e_i  = ncm_vector_get (self->weights, i) *_ncm_stats_dist_nd_kde_studentt_studentt(self, (m2lnp_i/ self->href2));
+    e_i  = ncm_vector_get (self->weights, i) * _ncm_stats_dist_nd_kde_studentt_studentt (self, (m2lnp_i / self->href2));
     t    = s + e_i;
     c   += (s >= e_i) ? ((s - t) + e_i) : ((e_i - t) + s);
     s    = t;
     /*printf ("% 22.15g % 22.15g % 22.15g\n", e_i, s, self->lnnorm);*/
   }
-printf("det: %lf \n norma:  %lf \n, resultado: %lf",ncm_matrix_cholesky_lndet (self->cov_decomp), self->lnnorm, log (s));
-  return -2.0 * (log (s) - self->lnnorm);
+  return -2.0 * (log (s) + self->lnnorm);
```

Comment:
This one is important. As I commented in the previous commit, I do not know whether to put + or - in the return. The calculation is surely with +, but the results are not good.